lib/compiler_rt/addtf3.zig

```diff
@@ -4,12 +4,14 @@ const addf3 = @import("./addf3.zig").addf3;
 pub const panic = common.panic;
 
 comptime {
-    if (common.want_ppc_abi) {
-        @export(__addkf3, .{ .name = "__addkf3", .linkage = common.linkage });
-    } else if (common.want_sparc_abi) {
-        @export(_Qp_add, .{ .name = "_Qp_add", .linkage = common.linkage });
-    } else {
-        @export(__addtf3, .{ .name = "__addtf3", .linkage = common.linkage });
+    if (common.should_emit_f80_or_f128) {
```

Comment:
They aren't trying to override the symbol. They're trying to prevent the compiler from outputting these symbols, as they produce invalid LLVM IR. See the issue thread and LLVM issue for context.